Build.bat

```diff
@@ -0,0 +1 @@
+tfx extension create --manifest-globs vss-extension.json
```

Comment:
Can you please amend the commit to remove the batch file from the commit instead of in a separate commit?